llvm/test/CodeGen/RISCV/cheri/compress.ll

```diff
@@ -0,0 +1,118 @@
+; This test is designed to run twice, once with function attributes and once
+; with target attributes added on the command line.
+; See compress.ll in the folder above
+;
+; RUN: cat %s > %t.tgtattr
+; RUN: echo 'attributes #0 = { nounwind }' >> %t.tgtattr
+; RUN: %riscv32_cheri_purecap_llc -mattr=+c,+xcheri,+cap-mode -filetype=obj < %t.tgtattr \
+; RUN:   | llvm-objdump -d -M no-aliases --syms - | FileCheck %s
+; RUN: %riscv64_cheri_purecap_llc -mattr=+c,+xcheri,+cap-mode -filetype=obj < %t.tgtattr \
+; RUN:   | llvm-objdump -d -M no-aliases --syms - | FileCheck %s
+; RUN: %riscv64_cheri_purecap_llc -mattr=+c,+xcheri,+cap-mode,+xcheri-norvc -filetype=obj < %t.tgtattr \
+; RUN:   | llvm-objdump -d -M no-aliases --syms - | FileCheck %s --check-prefix=NORVC
```

Comment:
CHECK and NORVC are a confusing set to use

Reply:
Gone for CHECK & CHECK-NORVC now.